src/lib/y2partitioner/setup_errors_presenter.rb

```diff
@@ -44,37 +44,51 @@ def initialize(setup_checker)
     #
     # @return [String]
     def to_html
```

Comment:
NP: I would simplify the method a little bit by defining a `#warnings_html` in a similar way to `#errors_html`:

```
def to_html
  errors_html || warnings_html || ""
end
```